### ISSUE TITLE
Filling the blanks does not work

### ISSUE THREAD

**Issue by dustinmoorenet** [Tom-Alexander/regression-js#29]
If I give a `null` value it does not correctly determine the equation.

```
$ node
> var regression = require('regression');
undefined
> regression('linear', [[0, 10], [1, null], [2, 10]]);
{ equation: [ 2.5, 5 ],
  points: [ [ 0, 5 ], [ 1, 7.5 ], [ 2, 10 ] ],
  string: 'y = 2.5x + 5' }
```

I expect:

```
{ equation: [ 0, 10 ],
  points: [ [ 0, 10 ], [ 1, 10 ], [ 2, 10 ] ],
  string: 'y = 0x + 10' }
```

**Comment by alltom** [comment]
I came across this too. Including a null value changes the regression.

**Comment by Tom-Alexander** [comment]
Fixed in #59
